gui/screens/settings.py

```diff
@@ -268,6 +268,16 @@ def __init__(self):
         super(EditAccountOrderSettings, self).__init__()
         self._account_item = QObject('settingsContentBaseScrollView_draggableDelegate_StatusDraggableListItem')
         self._accounts_list = QObject('statusDesktop_mainWindow')
+        self._text_item = QObject('settingsContentBaseScrollView_StatusBaseText')
+        self._back_button = Button('main_toolBar_back_button')
+
+    @property
+    @allure.step('Get all edit account order text labels')
+    def text_labels_from_edit_account_order_settings(self):
```

Comment:
Discussed in private chat and decided to rename it to get_account_recommendations, if it is a property - account_recommendations

Reply:
Done